src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -23,12 +23,22 @@ namespace Dynamo.Controls
     public partial class NodeView : IViewModelView<NodeViewModel>
     {
         public delegate void SetToolTipDelegate(string message);
-        public delegate void UpdateLayoutDelegate(FrameworkElement el);       
+
+        public delegate void UpdateLayoutDelegate(FrameworkElement el);
+
         private NodeViewModel viewModel = null;
         private PreviewControl previewControl = null;
         private const int previewDelay = 1000;
 
         /// <summary>
+        /// ZIndex is used to order nodes, when some node is clicked.
+        /// This selected node should be moved above others.
+        /// Start value of zIndex is 3, because 1 is for groups and 2 is for connectors.
+        /// Nodes should be always at the top.
+        /// </summary>
+        private static uint zIndex = 3;
```

Comment:
Why can't this be int? Int32.Maxvalue is sufficient here.  uint is not cls complaint and I don't see any specific use of uint.